exec/java-exec/src/main/resources/rest/profile/profile.ftl

```diff
@@ -132,7 +132,7 @@
                 <#list model.getOptionList() as option>
                   <tr>
                     <td>${option.getName()}</td>
-                    <td>${option.getValue()?c}</td>
+                    <td>${option.getValue()?string}</td>
```

Comment:
Tried this out. All the options take non-null values.  Any attempt to set a property (String, for e.g.) to null failed. So, I guess as long as a property takes only non-null values, things should be fine.